app/Http/Controllers/API/ServerProviderController.php

```
@@ -49,7 +50,10 @@ public function create(Request $request, Project $project): ServerProviderResour
         $this->authorize('create', ServerProvider::class);
 
         $user = user();
-        $serverProvider = app(CreateServerProvider::class)->create($user, $request->all());
+
+        abort_unless(TokenProjectScope::canCreate($user, $project->id, $request->boolean('global')), 403);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Support\TokenProjectScope' in method 'create'. (undefined)

(StaticAccess)